src/main/java/org/fenixedu/academic/domain/phd/SearchPhdIndividualProgramProcessBean.java

```diff
@@ -263,7 +265,10 @@ public boolean test(PhdIndividualProgramProcess toEval) {
             result.add(new InlinePredicate<PhdIndividualProgramProcess, String>(getName()) {
                 @Override
                 public boolean test(PhdIndividualProgramProcess toEval) {
-                    return Person.findPerson(getValue()).contains(toEval.getPerson());
+                    String normalizedName = StringNormalizer
+                            .normalizeAndRemoveAccents(toEval.getPerson().getProfile().getFullName().toLowerCase().trim());
```

Comment:
Here I'm assuming that *every* `PhdIndividualProgramProcess` is associated to a `Person`. Please correct me if I'm wrong.